extensions/ql-vscode/src/databases/ui/db-tree-view-item-action.ts

```diff
@@ -59,3 +59,14 @@ function canBeRenamed(dbItem: DbItem): boolean {
 function canBeOpenedOnGitHub(dbItem: DbItem): boolean {
   return dbItemKindsThatCanBeOpenedOnGitHub.includes(dbItem.kind);
 }
+
+export function getGitHubUrl(dbItem: DbItem): string | undefined {
```

Comment:
I _could_ have put this logic directly in `db-panel.ts`, but this made it slightly easier to test 😄

Reply:
Since this method method is called get `Url` we could return an url directly (instead of a string), so that the caller doesn't have to care about conversion. 

It might make sense (not sure) to put it in `extensions/ql-vscode/src/common/github-url-identifier-helper.ts`?

Reply:
WRT location, I think where it is makes sense - it's a function that helps us with db tree view item actions. It's not generic enough to be in `common` since it's specific to `DbItem`.

Reply:
> Since this method is called get `Url` we could return an url directly (instead of a string), so that the caller doesn't have to care about conversion.

After a bit of [experimentation](https://github.com/github/vscode-codeql/pull/1921#discussion_r1061332949), turns out the `vscode.open` command wanted a Uri or string, so I've kept this as-is!